testing/trino-product-tests/src/main/java/io/trino/tests/product/BaseTestTableFormats.java

```diff
@@ -121,13 +121,21 @@ protected void testPathContainsSpecialCharacter(String schemaLocation, String pa
                     row(11, "with%%percents"),
                     row(12, "with space"));
             assertThat(onTrino().executeQuery(format("SELECT * FROM %1$s.test.%2$s", getCatalogName(), tableName))).containsOnly(expectedRows);
+            if (testSparkCompatibility()) {
```

Comment:
Can we leave a code comment why this condition is required? It's not clear when reading the code.

Reply:
ya will do. the delta container is not configured in this env for spark. We will have to either create a separate env for testing delta or have a consolidated image for spark iceberg and delta

Reply:
Thanks for your explanation. I get it now. Then, how about using the existing `getCatalogName` method?

Reply:
done

Reply:
> I guess there are tests already for spark compat in delta-oss , so not a big deal if we skip

We need compatibility tests between Trino and Spark on Azure & GCS to ensure that what we write is actually readable, and hopefully correct,  by Spark.

>  the delta container is not configured in this env for spark. 

@anusudarsan  Can you please add a follow-up to address this point?